packages/firestore/src/remote/backoff.ts

```diff
@@ -31,6 +31,7 @@ const LOG_TAG = 'ExponentialBackoff';
 export class ExponentialBackoff {
   private currentBaseMs: number;
   private timerPromise: CancelablePromise<void> | null = null;
+  private lastAttempt: Date = new Date(0);
```

Comment:
Since the "current time" is not a portably expressed, maybe make this lastAttemptTime stored as a number?

That way we don't have two different ways of reading the current time in the code (currently `new Date().getTime()` vs `new Date()`).

Reply:
Renamed to lastAttemptTime and switched to epoch milliseconds.